app/jobs/api/issues_unlabel_issue_job.rb

```diff
@@ -2,9 +2,17 @@ module Api
   class IssuesUnlabelIssueJob < IssueEventJob
     timestamp ->(params) { params[:issue]['updated_at'] }
     action "issue_unlabeled"
+    cache_key ->(message) {
+      label_name = message[:payload][:label] ? message[:payload][:label]['name'] : ""
+
+      "#{message[:meta][:action]}.#{message[:meta][:user]["login"]}.#{message[:meta][:identifier]}.#{message[:meta][:timestamp]}.#{ label_name }"
```

Comment:
I think what he's saying is that there's a lot of overlap between cache keys in general, and an exact match between labeled/unlabeled keys.